core/src/validator.rs

```diff
@@ -1322,7 +1340,7 @@ impl Validator {
             config.wait_to_vote_slot,
             accounts_background_request_sender,
             config.runtime_config.log_messages_bytes_limit,
-            &connection_cache,
+            &tpu_connection_cache, // for the cache warmer
```

Comment:
The other one does not need to be warmed?

Reply:
> The other one does not need to be warmed?

We never implemented cache warming for tpu forward anyway. So this is not a new difference.